templates/commands/render.go

```diff
@@ -17,19 +17,27 @@ package commands
 
 import (
 	"context"
+	"crypto/rand"
+	"encoding/binary"
 	"errors"
 	"flag"
 	"fmt"
 	"io/fs"
 	"os"
+	"path/filepath"
 
 	"github.com/abcxyz/pkg/cli"
+	"github.com/abcxyz/pkg/logging"
+	"github.com/hashicorp/go-getter/v2"
 )
 
+var logger = logging.NewFromEnv("ABC_")
```

Comment:
I'm not a fan of making this global, since overriding it for tests is then harder. Can we embed it in the command or context instead?

Reply:
Sure, moved to a field on the Render struct.

Reply:
Now moved to context per the other thread.